### PR TITLE
Feature/express with multer

### PR DESCRIPTION
**Pull request by vc-jinal**
Created by Mohit

### CODE REVIEW

File: express-crud-multer/employee.router.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    if (getEmployee) {
+        res.json(getEmployee);
+    } else {
+        res.status(404).json({ error: "Employeee not Found..!!!" })
+    }
+})
+
+//create emplyee details
+employeeRouter.post('/', fileUpload.singleUpload, async (req, res) => {
+    console.log('bcejebcjbhbhcbdhcbh------');
```
Comment (vc-mohit) [inline express-crud-multer/employee.router.js:27]: remove console

Comment (vc-jinal) [reply express-crud-multer/employee.router.js]: console removed

File: express-crud-multer/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "body-parser": "^1.20.2",
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
+    "multer": "^1.4.5-lts.1",
+    "nodemon": "^2.0.22"
```
Comment (vc-mohit) [inline express-crud-multer/package.json:17]: it should in dev dependency.

File: express-crud-multer/employee.router.js (hunk 1 above)
Comment (vc-jinal) [reply express-crud-multer/employee.router.js]: remove consoles

File: express-crud-multer/package.json (hunk 2 above)
Comment (vc-jinal) [reply express-crud-multer/package.json:19]: move nodemon dependencies to dev Dependencies

**Review by vc-mohit — APPROVED** [review]

**Pull request merged**